src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -239,11 +239,11 @@ AthensSurfaceExamples class >> example3 [
 			can pathTransform restoreAfter: [
 				can pathTransform rotateByDegrees: i*5.
 				can pathTransform scaleBy: (1 - (i/2000)).
-				can drawShape: path ].
-			
-			Display getCanvas drawImage: surf asForm  at: 0@0. 
+				can drawShape: path ].			
 		]	
-	]
+	].
+
+	surf asForm asMorph openInWindow. 
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?